web3/utils/normalizers.py

```diff
@@ -124,3 +129,28 @@ def abi_ens_resolver(w3, abi_type, val):
     addresses_checksummed,
     decode_abi_strings,
 ]
+
+
+# ----- Property Normalizers -----
```

Comment:
*nitpick*

I don't know it it's actually present in this codebase, but style-wise, for comment headings I prefer

```
#
# The Heading
#
```

I find it provides an easier to spot visual indicator that a new logical section is beginning.